docs/README.md

```diff
@@ -1,27 +1,28 @@
 # KubeAI: AI Inferencing Operator
 
-The easiest way to serve ML models in production. Supports LLMs, embeddings, and speech-to-text.
-
-✅️  OpenAI API Compatibility: Drop-in replacement for OpenAI  
-⚖️  Autoscaling: Scale from zero, autoscale based on load  
-🧠  Serve text generation models with vLLM or Ollama  
-🔌  Dynamic LoRA adapter loading  
-⛕  Inference-optimized load balancing  
-💬  Speech to Text API with FasterWhisper  
-🧮  Embedding/Vector API with Infinity  
-🚀  Multi-platform: CPU, GPU, TPU  
-💾  Model caching with shared filesystems (EFS, Filestore, etc.)  
-🛠️  Zero dependencies (does not depend on Istio, Knative, etc.)  
-💬  Chat UI included ([OpenWebUI](https://github.com/open-webui/open-webui))  
-✉  Stream/batch inference via messaging integrations (Kafka, PubSub, etc.)  
+Deploy and scale machine learning models on Kubernetes. Built for LLMs, embeddings, and speech-to-text.
+
+## Key Features
+
+🚀 **LLM Operator** - Manages vLLM and Ollama servers  
+🔗 **OpenAI Compatible** - Works with OpenAI client libraries  
+🛠️ **Simple Deployment** - No external depencies required  
+⚡️ **Intelligent Scaling** - Scale from zero to meet demand  
+⛕ **Smart Routing** - LLM-specific load balancing algorithms  
+🧩 **Dynamic LoRA** - Hot-swap model adapters with zero downtime  
+🖥 **Hardware Flexible** - Runs on CPU, GPU, or TPU  
+💾 **Efficient Caching** - Supports EFS, Filestore, and more  
+🎙️ **Speech Processing** - Transcribe audio via FasterWhisper  
+🔢 **Vector Operations** - Generate embeddings via Infinity  
```

Comment:
Rename to Embedding Operator - manages Infinity servers.

I think Vector Operations isnt what I would think of when think of when looking for vector or embedding server.